lib/govuk_publishing_components/presenters/contextual_navigation.rb

```diff
@@ -114,7 +114,77 @@ def show_brexit_related_links?
-        content_item["content_id"] == "c3752802-f091-43a9-ba90-33568fccf391"
+        # /government/news/30-creative-teams-awarded-up-to-100000-each-for-festival-uk-2022-rd-project
+        document_exceptions = %w[
+          c3752802-f091-43a9-ba90-33568fccf391
+        ]
+
+        document_type_exceptions = %w[
+          aaib_report
+          answer
+          asylum_support_decision
+          fatality_notice
+          maib_report
+          raib_report
+          simple_smart_answer
+          transaction
+        ]
+
+        # Entering and staying in the UK
+        # Going and being abroad
+        # Corporate information
+        # Transport
+        # Environment
+        # International
+        # Defence and armed forces
+        # Society and culture
+        # Government
+        # Work
+        # Welfare
+        # Money
+        # Business and industry
+        # Health and social care
+        # Education > Further and higher education
+        # Education > Teaching and leadership
+        # Education > Funding and finance for students
+        # Coronavirus (COVID-19)
+        taxon_allow_list = %w[
+          ba3a9702-da22-487f-86c1-8334a730e559
+          9597c30a-605a-4e36-8bc1-47e5cdae41b3
+          a544d48b-1e9e-47fb-b427-7a987c658c14
+          a4038b29-b332-4f13-98b1-1c9709e216bc
+          3cf97f69-84de-41ae-bc7b-7e2cc238fa58
+          37d0fa26-abed-4c74-8835-b3b51ae1c8b2
+          e491505c-77ae-45b2-84be-8c94b94f6a2b
+          e2ca2f1a-0ff3-43ce-b813-16645ff27904
+          e48ab80a-de80-4e83-bf59-26316856a5f9
+          d0f1e5a3-c8f4-4780-8678-994f19104b21
+          dded88e2-f92e-424f-b73e-6ad24a839c51
+          6acc9db4-780e-4a46-92b4-1812e3c2c48a
+          495afdb6-47be-4df1-8b38-91c8adb1eefc
+          8124ead8-8ebc-4faf-88ad-dd5cbcc92ba8
+          dd767840-363e-43ad-8835-c9ab516633de
+          ff00b8b2-dcdb-4659-93c2-291c9be354f3
+          23265b25-7ec3-4960-8517-4ff8d4d92cac
+          5b7b9532-a775-4bd2-a3aa-6ce380184b6c
+        ]
+
+        # Entering and staying in the UK > Immigration offences
+        # Entering and staying in the UK > Inspections of border, immigration and asylum services
+        # Entering and staying in the UK > Refugees, asylum and human rights
+        # Transport > Local transport
+        # Education > Further and higher education > Education in prisons
+        taxon_exceptions = %w[
+          fa13521f-9285-45b0-bd65-4a472a8037e7
+          0fab9131-f877-4286-b4d8-922fbfb402b6
+          08a8a69f-2825-4fe2-a4cf-c83458a5629e
+          3b4d6319-fcef-4637-b35a-e3df76321894
+          6426d1c5-93c8-4659-85d5-1f0d3368a124
+        ]
+
+        document_exceptions.include?(content_item["content_id"]) ||
+          document_type_exceptions.include?(content_item["document_type"]) ||
+          content_item.dig("links", "taxons").to_a.any? { |taxon| taxon_exceptions.include?(taxon["content_id"]) && !taxon_allow_list.include?(taxon["content_id"]) }
```

Comment:
Should either or both of the taxon_allow_list/taxon_exceptions lists take into account the full tree of taxons, and not just the ones they're directly tagged to?

For example, https://www.gov.uk/api/content/government/news/transport-secretary-launches-794-million-investment-to-boost-rail-links-in-north-and-south is tagged to `transport/rail-infrastructure`, and therefore indirectly `transport`. This won't be reflected here unless we traverse down the tree of linked taxons.

Reply:
Good point, Simon. Will update to look at the full taxon tree.